zebedee/client.go

```diff
@@ -306,6 +306,38 @@ func (c *Client) GetPageTitle(ctx context.Context, userAccessToken, collectionID
 	return pt, nil
 }
 
+// GetPageType retrieves the type of content and page title for a given URI from zebedee
+func (c *Client) GetPageType(ctx context.Context, userAccessToken, collectionID, lang, uri string) (PageType, error) {
+	reqURL := c.createRequestURL(ctx, collectionID, lang, "/data", "uri="+uri+"&type")
+	b, _, err := c.get(ctx, userAccessToken, reqURL)
+	if err != nil {
+		return PageType{}, err
+	}
+
+	var pt PageType
+	if err = json.Unmarshal(b, &pt); err != nil {
+		return pt, err
+	}
+
+	return pt, nil
+}
+
+// GetPageTypeAndMetadata retrieves the type of content and page title for a given URI from zebedee
+func (c *Client) GetPageTypeAndMetadata(ctx context.Context, userAccessToken, collectionID, lang, uri string) (PageTypeAndMetadata, error) {
+	reqURL := c.createRequestURL(ctx, collectionID, lang, "/data", "uri="+uri+"&type")
+	b, _, err := c.get(ctx, userAccessToken, reqURL)
+	if err != nil {
+		return PageTypeAndMetadata{}, err
+	}
+
+	var pt PageTypeAndMetadata
+	if err = json.Unmarshal(b, &pt); err != nil {
+		return pt, err
+	}
+
+	return pt, nil
+}
+
```

Comment:
I feel like I"m missing something here - these look identical.

Reply:
`GetType` returns just the type and `GetPageTypeAndMetadata` returns type and some metadata. Was using type first, then realised we needed more data so added a new func and left the old one. Happy to remove `GetType` if we think it's superfluous

Reply:
As per our call i've remove both funcs and replaced them with a `GetPageData` func that returns a more generic page response